docs/reference-guides/interactivity-api/iapi-faq.md

```diff
@@ -0,0 +1,129 @@
+- **Composability and compatibility**: You can combine interactive blocks, nest them in structures with defined behaviors, and, thanks to following the same standard, they are fully cross-compatible. If each block were to use a different approach to interactivity, they would likely break.
+- **Fewer KBs will be sent to the browser**. If each plugin author uses a different JS framework, more code will be loaded in the frontend. If all the blocks use the same one, the code is reused.
+- If all the blocks on a page use this standard, site-wide features like client-side navigation can be enabled.
+
+## What are the performance implications of using this API? Is it worth loading the Interactivity API for very simple use cases?
+
+The API has been designed with performance in mind, so it shouldn’t be a problem:
+
+- **The runtime code needed for the directives is just ~10 KB**, and it only needs to be loaded once for all the blocks.
+- **It only loads the directives needed** by the blocks present on the page. For example, if no blocks use data-wp-show, that directive won’t be loaded.
+- **All the scripts that belong to the Interactivity API (including the `view.js` files) will load without blocking the page rendering.**
+- There are ongoing explorations about the possibility of **delaying the scripts loading once the block is in the viewport**. This way, the initial load would be optimized without affecting the user experience.
+
+## Can I use directives in the block editor?
+
+No. Right now, directives only work in the frontend of blocks. However, it’ll be investigated whether some directives (and also your custom directives) could be reused across the frontend and the editor. It’s worth emphasizing that the realities of the editor application and the frontend of a website are very different, particularly around the interactivity they afford. It needs to be ensured that the right tools are built for the right context. An interesting area to explore further would be to expose directives in the editor so users and builders can use them to attach behaviors to their sites on demand.
+
+## Does it work with the Core Translation API?
+
+It does! As the Interactivity API works perfectly with server-side rendering, you can use all the WordPress APIs including [`__()`](https://developer.wordpress.org/reference/functions/__/) and [`_e()`](https://developer.wordpress.org/reference/functions/_e/). You can use it to translate the text in the HTML (as you normally would) and even use it inside the store when using `wp_initial_state()` on the server side. It might look something like this:
+
+```php
+// render.php
+wp_interactivity_state( 'favoriteMovies', array(
+      "1" => array(
+        "id" => "123-abc",
+        "movieName" => __("someMovieName", "textdomain")
+      ),
+) );
+```
```

Comment:
This section seems misleading. Static strings in the interactive HTML can be rendered in PHP via the Translation API, but JavaScript translations aren't really supported yet in Script Modules. https://core.trac.wordpress.org/ticket/60234

Reply:
We showcased the translation support in a WCEU. So I would keep it, but also explain how to set translatable JS strings and mention that would be improved.

Reply:
If I understand this properly, at this stage, translations within the Interactivity API are available on the server functions (PHP) but **still** not on the client methods (Javascript). 

I have adjusted the explanation and added the following at the end of the response:

> A translation API compatible with script modules (needed for the Interactivity API) is 
currently being worked on. Check https://core.trac.wordpress.org/ticket/60234 to follow the progress on this work.

Reply:
For non-interactive translated texts, you can just add them in PHP.

```php
<span><?php _e('hello'); ?></span>
```

For interactive translated texts, you can add the strings of translated text to the state in PHP and then use them on the client side:

```php
wp_interactivity_state('...', array(
  'hello' => __('hello'),
  'bye' => __('bye'),
  'welcomeText' => __('hello') // initial value of derived state for SSR
));
```

```html
<span data-wp-text="state.welcomeText"></span>
```

```js
store("...", {
  state: {
    get welcomeText() {
      return state.isUserLeaving ? state.bye : state.hello;
    },
  },
});
```

What is harder to do is things like plurals, numbers and such...